# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/browser.js
# Commit: 5f48ef706159
# Full Hash: 5f48ef706159275c966c80b13cf09d7dcc1db932
# Author: Christoph Walcher <christoph-wa@gmx.de>
# Date: 2019-06-15 21:44:24
# Regressor Bug: 909760
# File Overlap Count: 1
# Description:
#   Bug 909760 - Show download progress in the MacOS Finder r=spohl,mak
#   
#   Show download progress in the MacOS Finder
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30688
# ==============================================================================

diff -r fc8f53ac1fa2 -r 5f48ef706159 browser/base/content/browser.js
--- a/browser/base/content/browser.js	Sat Jun 15 11:40:02 2019 +0000
+++ b/browser/base/content/browser.js	Sat Jun 15 15:46:36 2019 +0000
@@ -1946,6 +1946,10 @@
         DownloadsCommon.initializeAllDataLinks();
         ChromeUtils.import("resource:///modules/DownloadsTaskbar.jsm", {})
           .DownloadsTaskbar.registerIndicator(window);
+        if (AppConstants.platform == "macosx") {
+          ChromeUtils.import("resource:///modules/DownloadsMacFinderProgress.jsm")
+            .DownloadsMacFinderProgress.register();
+        }
       } catch (ex) {
         Cu.reportError(ex);
       }